assign_rights/tests.py

```diff
@@ -26,6 +26,7 @@ def setUp(self):
         self.api_factory = APIRequestFactory()
         self.factory = RequestFactory()
         self.user = User.objects.create_user("test_user", "test@example.com", "testpass")
+        self.group = Group.objects.get(name='edit').user_set.add(User.objects.get(username='test_user'))
```

Comment:
This is a little byzantine. How about:
```suggestion
        self.group = Group.objects.get(name='edit')
        self.user.groups.add(self.group)
```
Alternatively, if you don't need to set the group to a variable you can also just do 
```suggestion
        self.user.groups.add(Group.objects.get(name='edit'))
```